Assignment icon updates

bendansby opened the pull request:
Updates and adds one new icon to AssignmentIcon

bcardiff commented:
@bendansby many (or all) of the Circled icons that are updated in this PR are loosing the benefits of https://github.com/NoRedInk/noredink-ui/pull/1575 since they are no longer using the `encircle` function.

I also noticed some clipPaths added, but no element with that id is defined. So I am not sure if things are looking as you would expect.

bendansby commented:
Thanks for the reminder about Alex's PR; I had forgotten that cleanup was needed there. I fixed the slightly off dimensions and replaced the paths with circles. The clipping paths are just an artifact of groups in Figma; looks fine but I went ahead and removed them for cleanliness.
